EIPS/eip-4762.md

```diff
@@ -251,7 +251,16 @@ None of:
 
 are warm at the start of a transaction.
 
-Note: When (and only when) calling a system contract _via a system call_, the code chunks and account should not appear in the witness.
+### System contracts
+
+When (and only when) calling a system contract either
+
+ * _via a system call_ or
+ * _to resolve a precompile/opcode_,
+
+ the system contract code chunks and account accesses should not appear in the witness as these should be known/cached in the clients. However any other accesses and all writes should appear in the witness.
+
+Also Corresponding witness costs need to be charged for _precompile/opcode resolution_ but are not charged in the _system call_.
```

Comment:
Is this clarification needed? As in, who would could charge for system call calls?

Reply:
i slightly favor having this for completeness and context sake.

Reply:
Fine to me. Just one extra nit: `s/Corresponding/corresponding`